room.py

```diff
@@ -26,8 +25,11 @@
     paylink,
     wall_metadata,
 )
-import url1.special.all
+import url1.special.all, url1.special.page, url1.movie.movieimages, url1.special.allbin
 
-from url2 import assets, reginfo
+from url2 import assets, reginfo, related
 
 from url3.pay import category_header, event_today, wall_metadata
+
+import models
```

Comment:
Please rebase your branch to what's currently pushed instead of overwriting changes. `import models` must be where it is in order to initialize the app and create within the DB if necessary.